src/ActionModal/components/ActionModal.vue

```diff
@@ -685,16 +691,14 @@ export default {
     },
     // limit fees to the maximum the user has
     adjustFeesToMaxPayable() {
-      if (this.invoiceTotal > this.selectedBalance.amount) {
```

Comment:
why did you move this check to `invoiceTotal`?

Reply:
because this.invoiceTotal doesn't exist yet. 

I mean, I could have done the `Number(this.subTotal) + this.estimatedFee >
        this.selectedBalance.amount` in here, true. But I thought this way is much more readable and also it's clear the hack against negative gasPrices will be always run